### DISCUSSION TITLE
Serious SEO issue when using Server Side Rendering - Duplicate Structured data being picked up by Google - please help!

### DISCUSSION THREAD

**Discussion by valley-boy** [vercel/next.js#67304] [Help]
### Summary

Apologies, a rather dramatic discussion title!

However we have hit, Have reached out to various contributors to no avail. Its a showstopper for our current use case, and if no solution we will need to migrate the website to a different framework, sadly..

All details on this ticket:

https://github.com/vercel/next.js/discussions/66896

Any feedback hugely appreciated

cheers, Oli

**Comment by c0b41** [comment]
duplicate https://github.com/vercel/next.js/issues/40143

**Reply by valley-boy** [reply]
ah apologies I missed this one, a great thread. Even more google indexing issues I had not foreseen. __NEXT_DATA__  makes it hard to argue the case for next.js SSR as an SEO acceptable solution in its current state. A real shame but really no choice but to explore the likes of astro.js

**Comment by JungfruPrast** [comment]
Hi, sorry for being late. I got the same issue for the duplicated structured data and found that using next/scripts afterinteractive strategy solved the duplication issue.
